### DISCUSSION TITLE
Include a config file with iPXE

### DISCUSSION THREAD

**Discussion by CyberC4p0** [ipxe/ipxe#819] [Q&A]
Hello to all! I am trying to PXE boot [WipeDrive Standalone ISO](https://www.wipedrive.com/wipedrive-enterprise-downloads/). I have the following config file:

```shell
#!ipxe

set wipedrive tftp://192.168.1.231/os-images/WipeDrive

kernel ${wipedrive}/isolinux/vmlinux    vmlinux
initrd ${wipedrive}/isolinux/initrd     initrd
imgargs vmlinux initrd=initrd boot=live netboot=nfs ip=dhcp nfsroot=${wipedrive} quiet splash ---

boot || goto failed
```

I would like to include the wd-options.cfg file when PXE booting WipeDrive. This file holds configurations for the wipe. These [instructions](https://support.wipedrive.com/support/solutions/articles/73000175723-how-to-create-a-custom-build) point me in the right direction but instead of flashing it on a USB, I would like to PXE boot the individual files. If any additional information is needed, please let me know. Your help with this is deeply appreciated!

**Comment by NiKiZe** [comment]
What you have there is an iPXE script.
Not sure what it is you want, but of you want iPXE with embedded script you can read more at https://ipxe.org/embed

It might also be that you want this .cfg file of yours to be available after you boot, if so then initrd with additional name can get it added to your cpio archive (initrd) but then having that distro read that file is not always the easiest. You should probably reach out to them, unless you are willing to start disecting their boot scripts.

**Reply by CyberC4p0** [reply]
Understood. I'll reach out to WhiteCanyon. Thank you for the info :)
